sme_finance_application_schema/entity_v1

```diff
@@ -46,6 +46,10 @@
             "description":"The average monthly revenue of the SME, in GBP",
             "type": "integer"
         },
+        "activities": {
+            "description":"Free text. Activities the company performs",
```

Comment:
Let's go broader here. Free text about the entity, not just activities it performs?